distributions/otelcol-otlp/otelcol-otlp.conf

```diff
@@ -0,0 +1,5 @@
+# Systemd environment file for the otelcol-otlp service
+
+# Command-line options for the otelcol-otlp service.
+# Run `/usr/bin/otelcol-otlp --help` to see all available options.
```

Comment:
Should we add a comment here telling the user to create the configuration file in the `/etc/otelcol-otlp/config.yaml` file?